src/commands/distribute/release.ts

```diff
@@ -62,6 +62,10 @@ export default class ReleaseBinaryCommand extends AppCommand {
   @longName("silent")
   public silent: boolean;
 
+  @help("Whether the release is mandatory for the testers (Not used for stores)")
```

Comment:
Since this does not take in an argument, maybe reword as "make this release mandatory for testers". One way or another, make it clear that the default is `false`. :)